portfolio/src/main/webapp/script.js

```diff
@@ -91,6 +91,73 @@ function makeElement(nickname, message) {
   document.getElementsByClassName("comment-section")[0].appendChild(newComment);
 }
 
-function DeleteComments(){
-  fetch("/delete-comment", {method: 'POST'}).then(response => FetchComments());
+function deleteComments() {
+  fetch("/delete-comment", { method: 'POST' }).then(response => fetchComments());
+}
+
+function loadUserInformation() {
```

Comment:
Sorry maybe the name is not good now since you have a /user-info endpoint now that you do not call here :) maybe rename it to loadAuthInformation?